values.schema.json

```diff
@@ -0,0 +1,479 @@
+                        "name": {
+                            "type": "string"
+                        },
+                        "key": {
+                            "type": "string"
+                        }
+                    }
+                }
+            }
+        },
+        "namespaceWhitelist": {
+            "type": "array",
+            "description": "DEPRECATED: Use multiple workspace providers instead"
+        },
+        "ssh": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "enable": {
+                    "type": "boolean",
+                    "description": "Enables accessing environments via SSH.",
+                    "default": true
+                }
+            }
+        },
+        "serviceTolerations": {
+            "$ref": "#/definitions/kube_tolerations"
+        },
+        "logging": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "human": {
+                    "description": "Where to send logs that are formatted for readability by a human. Set to an empty string to disable.",
+                    "type": "string",
+                    "default": "/dev/stderr"
+                },
+                "stackdriver": {
+                    "description": "Where to send logs that are formatted for Google Stackdriver. Set to an empty string to disable.",
+                    "type": "string",
+                    "default": ""
+                },
+                "json": {
+                    "description": "Where to send logs that are formatted as JSON. Set to an empty string to disable.",
+                    "type": "string",
+                    "default": ""
+                },
+                "splunk": {
+                    "type": "object",
+                    "additionalProperties": false,
+                    "description": "Coder can send logs directly to Splunk, in addition to file-based output, if these values are configured. The channel is optional, and this logging is disabled if either the URL and Token are set to the empty string.",
+                    "properties": {
+                        "url": {
+                            "type": "string"
+                        },
+                        "token": {
+                            "type": "string"
+                        },
+                        "channel": {
+                            "type": "string"
+                        }
+                    }
+                }
+            }
+        },
+        "deploymentAnnotations": {
+            "type": "object"
+        }
+    },
+    "definitions": {
+        "kube_resource": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "storage": {
+                    "$comment": "Can this also be a number? Add regex to the strings to be valid resource strings",
```

Comment:
Not sure if numbers are supported, but we should just keep it string only anyways to avoid ambiguity